rskj-core/src/main/java/co/rsk/config/RemascConfig.java

```diff
@@ -30,7 +30,9 @@
     // Number of blocks block reward is split into
     private long syntheticSpan;
 
-    // RSK labs address
+    // RSK labs address.
+    // Note that his has to be a basic type (such as String) because RemascConfig
+    // is deserialized automatically from JSON.
     private String rskLabsAddress;
```

Comment:
We can define a [Jackson Mixin](https://github.com/FasterXML/jackson-docs/wiki/JacksonMixInAnnotations) for the `RemascConfig` object and set it to the mapper in `RemascConfigFactory`. This is if we don't want to taint our model with Jackson annotations.

Reply:
Sounds amazing. It would be great to do that along the "Remove JsonSimple" task.